queries.sql

```diff
@@ -0,0 +1,17 @@
+/* Add Queries in-line with the requirements */
+
+SELECT * from animals where animal_name LIKE '%mon';
+
+SELECT animal_name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/12/31';
+
+SELECT animal_name FROM animals WHERE neutered = 'true' AND escape_attempts <= '3';

```

Comment:
Please list the name of all animals that have less than 3 escape attempts not equal to and less than 3 attempts.

Reply:
Thanks for the feedback, issues have been resolved